modules/402-ingress-nginx/templates/controller/controller.yaml

```
@@ -334,6 +338,10 @@ spec:
         - mountPath: /chroot/etc/nginx/webhook-ssl/
           name: webhook-cert
           readOnly: true
+        {{- if $geoIP2.maxmindLicenseKey }}
+        - mountPath: /chroot/etc/ingress-controller/geoip/
```

```suggestion
        - mountPath: /chroot/etc/nginx/geoip/
```
Why special directory?

---

Same directory is used by ingress nginx controller for initial download.